### PR TITLE
Drive

### PR DESCRIPTION
**Pull request by liaarbel**

### CODE REVIEW

File: src/main/java/frc/team3388/robot/RobotMap.java [hunk 1]
```diff
@@ -14,4 +14,9 @@
     public static final int PISTON_REVERSE_CHANNEL = 2;
     public static final int PISTON_FORWARD_CHANNEL = 3;
 
+    // drive motors
+    public static final int RIGHT1 = 1;
```
Comment (tomtzook) [inline src/main/java/frc/team3388/robot/RobotMap.java]: Add `DRIVE_` prefix to all of the constants here. It will help recognize what this means.

**Review by tomtzook — CHANGES_REQUESTED** [review]
Nice work.
Almost.

File: src/main/java/frc/team3388/robot/RobotMap.java (hunk 1 above)
Comment (liaarbel) [reply src/main/java/frc/team3388/robot/RobotMap.java]: fixed

**Review by tomtzook — APPROVED** [review]
nice work

**Pull request merged**